packages/components/src/SegmentedControl/SegmentedControl.module.css

```diff
@@ -3,13 +3,17 @@
   grid-template-columns: repeat(var(--segmentedControl--option-count), 1fr);
   align-items: center;
   position: relative;
-  min-height: 32px;
+  min-height: 40px;
   box-sizing: border-box;
   border: var(--border-base) solid var(--color-border--interactive);
   border-radius: var(--radius-base);
   background-color: var(--color-surface--background);
 }
 
+.small {
+  min-height: 32px;
```

Comment:
I don't know if we have an opinion on this, but would it make sense to follow the pattern from Button and use a CSS var in these `min-height:` property declarations, so we're only manipulating a variable instead of setting new declarations?

![image](https://github.com/user-attachments/assets/942758ca-086d-4a1d-bd0f-d310e9c88291)
![image](https://github.com/user-attachments/assets/dbe0c0dc-bc8c-4885-a3b2-7163455c14a7)

Reply:
I debated using a variable but when I only had `small` and `base` I thought I'd leave it as is. I don't think we have a solid opinion on this as a team.

_But_ I also like the idea of just including `large` while we're here to align with our other components. So `large` was added here 6441414ba6dd73bf8d7d40548b0ca7ef019f0189 and then I cleaned up the css to match `Button` here e40927ed5fea2d320cdd1d51275ebc79739c6520